safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -77,6 +80,26 @@ def set_widgets(self):
             volcano_name_checkbox = QCheckBox(tr('Volcano Name'))
             volcano_name_line_edit = QLineEdit()
 
+            # Volcano longitude
+            volcano_longitude_checkbox = QCheckBox(
+                tr('Volcano longitude'))
+            volcano_longitude_spin_box = QDoubleSpinBox()
+            volcano_longitude_spin_box.setMinimum(-180)
+            volcano_longitude_spin_box.setMaximum(180)
+            volcano_longitude_spin_box.setSuffix(u' \xb0')  # degree symbol
+
+            # Volcano latitude
+            volcano_latitude_checkbox = QCheckBox(
+                tr('Volcano latitude'))
+            volcano_latitude_spin_box = QDoubleSpinBox()
+            volcano_latitude_spin_box.setMinimum(-90)
+            volcano_latitude_spin_box.setMaximum(90)
+            volcano_latitude_spin_box.setSuffix(u' \xb0')  # degree symbol
```

Comment:
`setSuffix('°')` is not working?

Reply:
It should be `setSuffix(u'°')`. Changed.